### PR TITLE
Closes DRUP8-906

### PR DESCRIPTION
**Pull request by emerham**
Add default metatags for sites both new and existing

### CODE REVIEW

**Review by mechanoid23 — APPROVED** [review]
This all looks good.
`metatag_firehose` gives me a chuckle.

I gave metatag defaults a close look. Similar to what I'm familiar with. Looks good.

I'm curious about your update_hook below.
Is it correct that while /config/install ymls are picked up on install, what you are doing here, is providing default config to already installed sites with this profile, on a db update?

**Comment by emerham** [comment]
metatag_firehose is the field type the module made, it is funny.

Yes that update hook adds the configs for existing sites. The Metatag Fields are not added to existing sites with this update hook. That can be achieved if need be.

File: config/install/core.entity_view_display.node.page.default.yml [hunk 1]
```diff
@@ -143,6 +145,21 @@ third_party_settings:
                 entity: layout_builder.entity
             weight: 2
             additional: {  }
+          23050431-0772-44a7-9c25-3a8047a3c59e:
+            uuid: 23050431-0772-44a7-9c25-3a8047a3c59e
```
Comment (Ruefa) [inline config/install/core.entity_view_display.node.page.default.yml:149]: should this uuid be here?

**Review by Ruefa — CHANGES_REQUESTED** [review]
Left a comment. Otherwise looks good

File: config/install/core.entity_view_display.node.page.default.yml (hunk 1 above)
Comment (emerham) [reply config/install/core.entity_view_display.node.page.default.yml:149]: Yeah that UUID is for the config type and needs to be there for layout builder. it's not the same as the "global" uuid for configs

**Pull request merged**